interface-definitions/include/firewall/add-dynamic-address-groups.xml.i

```diff
@@ -0,0 +1,49 @@
+<!-- include start from firewall/add-dynamic-address-groups.xml.i -->
+<leafNode name="address-group">
+  <properties>
+    <help>Dynamic address-group</help>
+    <completionHelp>
+      <path>firewall group dynamic-group address-group</path>
+    </completionHelp>
+  </properties>
+</leafNode>
+<leafNode name="timeout-value">
+  <properties>
+    <help>Set timeout</help>
+    <valueHelp>
+      <format>u32:1-255</format>
+      <description>Remains element in group for this time</description>
+    </valueHelp>
+    <constraint>
+      <validator name="numeric" argument="--range 1-255"/>
+    </constraint>
+  </properties>
+</leafNode>
+<leafNode name="timeout-unit">
+  <properties>
+    <help>Timeout unit: second/minute/hour/day</help>
```

Comment:
I don't think it's a good idea to duplicate formats in help strings, since tab completion already shows them. But see above: maybe we should use a different syntax instead of a separate unit command.